src/app/GitUI/Editor/Diff/DiffHighlightService.cs

```diff
@@ -311,32 +311,35 @@ private static IEnumerable<TextMarker> GetDifferenceMarkers(Func<int, char> getC
             int beforeLength = beginOffset - lineStartOffset;
             if (beforeLength > 0)
             {
-                yield return CreateUnmarker(lineRemoved.Offset + lineStartOffset, beforeLength);
+                yield return CreatePaleMarker(lineRemoved.Offset + lineStartOffset, beforeLength, GetRemovedBackColor());
             }
 
             int afterBegin = beginOffset + removedLength;
             int afterLength = lineRemoved.Length - afterBegin;
             if (afterLength > 0)
             {
-                yield return CreateUnmarker(lineRemoved.Offset + afterBegin, afterLength);
+                yield return CreatePaleMarker(lineRemoved.Offset + afterBegin, afterLength, GetRemovedBackColor());
             }
         }
         else
         {
             int length = lineRemoved.Length - lineStartOffset;
             if (length > 0)
             {
-                yield return CreateUnmarker(lineRemoved.Offset + lineStartOffset, length);
+                yield return CreatePaleMarker(lineRemoved.Offset + lineStartOffset, length, GetRemovedBackColor());
             }
         }
 
         yield break;
 
-        static TextMarker CreateUnmarker(int offset, int length)
-            => CreateTextMarker(offset, length, AppColor.AnsiTerminalWhiteBackBold.GetThemeColor());
+        static TextMarker CreatePaleMarker(int offset, int length, Color color)
+            => CreateTextMarker(offset, length, new HslColor(color).WithLuminosity(0.5 + (0.47 * (ColorHelper.IsLightTheme() ? 1 : -1))).ToColor());
```

Comment:
Add the color transformer to ColorHelper?

This is slightly brighter than the DimColor that can be set in the terminal. Should these be the same?
The Dim implementation is in AnsiUtilities right now, could be moved to ColorHelper.
Great change!

Reply:
> This is slightly brighter than the DimColor that can be set in the terminal.

I am going to try that. But I did not like it with lower luminosity.

Reply:
I have refactored it with `DimColor` and have applied it twice.